### PR TITLE
Add automatic load data when docker up

### PR DESCRIPTION
**Pull request by neungkl**
I think it is easier when docker automatic migrate all datas for you.

- [x] Tested on `docker-compose up --build`
- [x] Tested on `docker-compose down` and `docker-compose up`

### CODE REVIEW

**Comment by neungkl** [comment]
I just forget the case when your local migration is change, the docker will bug.

**Comment by neungkl** [comment]
Any reason? @futurizing

**Comment by futurizing** [comment]
If webserver's devs want to do something that changes schema, then run compose up again, the schema might not match with the content in `dump.json`.

File: webserver/docker_entrypoint.sh [hunk 1]
```diff
@@ -3,5 +3,11 @@
...
 
 ./wait_to_start.sh python3 manage.py migrate
-python3 manage.py loaddata dump.json
+python3 manage.py loaddata dump.json || {
+  echo ""
+  echo "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+  echo "  Can't dump data to database, maybe your migration was changed."
+  echo "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+  echo ""
+}
```
Comment (neungkl) [inline webserver/docker_entrypoint.sh]: Is it better? @futurizing @JThanat

**Comment by neungkl** [comment]
Can we discuss about this? @futurizing @JThanat

**Comment by futurizing** [comment]
I still have a feeling that we should do this manually.

**Comment by neungkl** [comment]
Consider to import data by yourself.

**Pull request closed without merging**